src/reanimated2/hook/useAnimatedRef.ts

```diff
@@ -31,11 +49,7 @@ export function useAnimatedRef<T extends ComponentRef>(): RefObjectFunction<T> {
     const fun: RefObjectFunction<T> = <RefObjectFunction<T>>((component) => {
       // enters when ref is set by attaching to a component
       if (component) {
-        tag.value = getTagValueFunction(
-          component.getScrollableNode
-            ? component.getScrollableNode()
-            : component
-        );
+        tag.value = getTagValueFunction(getScrollableRef(component));
```

Comment:
Can we also use `getScrollableRef` in `useAnimatedRef.ts`?

Reply:
I'm not sure what do you mean.